pkgs/development/python-modules/cleanlab/default.nix

```
@@ -44,13 +44,38 @@ buildPythonPackage {
     pandas
   ];
 
+  # This is ONLY turned off when we have testing enabled.
+  # The reason we do this is because of duplicate packages in the enclosure
+  # when using the packages in nativeCheckInputs.
+  # Affected packages: grpcio protobuf tensorboard tensorboard-plugin-profile
+  catchConflicts = (!doCheck);
+  doCheck = true;
```

```suggestion
  catchConflicts = !doCheck;
```

Checks are enabled by default in the python builders.

---

Yes, but since ``catchConflicts`` requires it to be defined, we must define it manually.
Otherwise, the derivation won't build with this error: ``error: undefined variable 'doCheck'``